src/vdom.rs

```diff
@@ -119,6 +121,21 @@ pub struct AppData<Ms: 'static, Mdl> {
     pub render_timestamp: Cell<Option<RenderTimestamp>>,
 }
 
+type OptDynRunCfg<Ms, Mdl, ElC, GMs> =
+    Option<AppRunCfg<Ms, Mdl, ElC, GMs, dyn builder::IntoAfterMount<Ms, Mdl, ElC, GMs>>>;
+
+pub struct AppRunCfg<Ms, Mdl, ElC, GMs, IAM: ?Sized>
+where
+    Ms: 'static,
+    Mdl: 'static,
+    ElC: View<Ms>,
+    IAM: builder::IntoAfterMount<Ms, Mdl, ElC, GMs>,
+{
+    mount_type: MountType,
+    into_after_mount: Box<IAM>,
+    phantom: std::marker::PhantomData<(Ms, Mdl, ElC, GMs)>,
```

Comment:
I suggest to add `use std::marker::PhantomData;` because `PhantomData` is a standard Rust type and to make it consistent with `builder.rs`.